### ISSUE TITLE
OutdatedDocumentation doesn't trigger when there is no param/property

### ISSUE THREAD

**Issue by Oshuremy** [detekt/detekt#6480]
Hi,

```
/**
* Test method
*/
fun test(param1: String, param2: String) : Boolean = true
```
-> This KDoc is valid for OutdatedDocumentation rule while no parameters are declared in the KDoc

```
/**
* Test method
*
* @param other
*/
fun test(param1: String, param2: String) : Boolean = true
```
-> This KDoc is not valid for OutdatedDocumentation rule

Why in my first case OutdatedDocumentation is OK ? How can I make it not ?

**Comment by lexa-diky** [comment]
From documentation: `If KDoc is not present or does not contain any @param or @property tags, rule violation will not be reported.`

I think this behavior was chosen because it is very common for documentation to not contain @param / @property list because argument names are already are good enough to understand and you don't really need any documentation

For example:
```kotlin
/**
 * Squares the number [n]
 * @param n number to be squared
**/
fun square(n: Int): Int == TODO()
```

But, it might be a good idea to have another rule checking that parameters are documented. 
Or even better this rule might be removed altogether, and replaced with set of more precise rules on documentation

**Comment by Oshuremy** [comment]
I'm agree with you, I thing those rules can be more precise to "force" same standards for anyone on the same project
